.expeditor/config.yml

```diff
@@ -49,13 +49,21 @@ pipelines:
         - ADHOC: true
 
 subscriptions:
+  - workload: artifact_published:unstable:chef-workstation:{{version_constraint}}
+    actions:
+      - built_in:build_docker_image
+  - workload: artifact_published:current:chef-workstation:{{version_constraint}}
+    actions:
+      - built_in:tag_docker_image
   - workload: artifact_published:stable:chef-workstation:{{version_constraint}}
     actions:
       - built_in:rollover_changelog
+      - bash:.expeditor/update_dockerfile.sh
+      - built_in:tag_docker_image
```

Comment:
Your actions here are out of order. `tag_docker_image` is a post-commit action, while bash actions are (by default) pre-commit actions. To address the expeditor config issue you'll want to move this action to the end of the list (or add a `post_commit: true`) to the subsequent bash actions.